scripts/tortoisesvn/tortoisesvn-install.reg.templ

```diff
@@ -366,15 +366,15 @@ REGEDIT4
 "{C5994568-53D9-4125-87C9-F193FC689CB2}"="TortoiseOverlays"
 
 [HKEY_LOCAL_MACHINE\SOFTWARE\TortoiseOverlays]
-"NormalIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\NormalIcon.ico"
-"ModifiedIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\ModifiedIcon.ico"
-"ConflictIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\ConflictIcon.ico"
-"DeletedIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\DeletedIcon.ico"
-"ReadOnlyIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\ReadOnlyIcon.ico"
-"LockedIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\LockedIcon.ico"
-"AddedIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\AddedIcon.ico"
-"IgnoredIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\IgnoredIcon.ico"
-"UnversionedIcon"="%TSVNHOME%\\TortoiseOverlays\\icons\\XPStyle\\UnversionedIcon.ico"
+"NormalIcon"="%TSVNHOME%\\Common\\TortoiseOverlays\\icons\\XPStyle\\NormalIcon.ico"
```

Comment:
I install Tortoisesvn and get `C:\Program Files\Common Files\TortoiseOverlays\icons\XPStyle\AddedIcon.ico` here. Is this the path to icon? If then, why we need to create a dir in C:\Program Files\Common Files\ ?

Reply:
I mean if we can change the path of icons, do we need to create a dir in  C:\Program Files\Common Files\ ? I wonder if it works without creating the symlink...

Reply:
Sorry for the late reply.

If we don't create symlink under `Program Files\Common Files`, it will not work properly.

I think `TortoiseSVN` does not handle all features of the overlay icons through that registry key. For some functions it still loads the icon files through `Program Files\Common Files`.

Reply:
Oh, I see...